### PR TITLE
TileLayer cache rendered sub layers

### PR DESCRIPTION
**Pull request by Pessimistress**
See discussion in https://github.com/uber/deck.gl/pull/3704

#### Change List
- Avoid excessive re-render during continuous viewport updates

### CODE REVIEW

**Comment by Pessimistress** [comment]
@jesusbotella

File: modules/geo-layers/src/tile-layer/utils/tile-cache.js [hunk 1]
```diff
@@ -11,23 +11,29 @@ export default class TileCache {
...
+    this._tiles = [];
 
-    if (maxZoom && parseInt(maxZoom, 10) === maxZoom) {
-      this._maxZoom = maxZoom;
+    if (Number.isFinite(maxZoom)) {
+      this._maxZoom = Math.floor(maxZoom);
     }
-    if (minZoom && parseInt(minZoom, 10) === minZoom) {
-      this._minZoom = minZoom;
+    if (Number.isFinite(maxZoom)) {
```
Comment (xintongxia) [inline modules/geo-layers/src/tile-layer/utils/tile-cache.js]: maxZoom -> minZoom

**Comment by jesusbotella** [comment]
I just wanted to say thank you! That was fast! 😀

**Comment by jererobles** [comment]
Is the cache configurable and if so should we add document it somewhere?

**Pull request merged**